### ISSUE TITLE
dont getting the beta version of steal

### ISSUE THREAD

**Issue by pYr0x** [stealjs/steal#825]
i cleaned my npm cache.

i want the steal-beta release installed.
in my package.json i have
```
"steal": "^1.0.0-beta",
```

but i always get steal `"_id": "steal@1.0.0-pre.7",`

https://semver.npmjs.com/ shows me that i should get the beta with requesting `^1.0.0-beta`.
@matthewp maybe a wrong publish on npm?

**Comment by cherifGsoul** [comment]
@pYr0x with:
```npm install steal@1.0.0-beta.3 --save-dev```
In my package.json I got:
```"steal": "^1.0.0-beta.3"

**Comment by cherifGsoul** [comment]
Actualy, we have 2 use cases with different results:
1- ```npm install steal@1.0.0-beta.3 --save-dev```installs the correct version
2- when we put the version manualy in package.json we get the pre version

**Comment by pYr0x** [comment]
found the problem!
@cherifGsoul at your frist install you requiring a `tag` release called `beta`. you will got the right version

the second installation, via package.json, you will receive the wrong version because:

http://semver.org/ said:
> Example: 1.0.0-alpha < 1.0.0. Precedence for two pre-release versions with the same major, minor, and patch version MUST be determined by comparing each dot separated identifier from left to right until a difference is found as follows: identifiers consisting of only digits are compared numerically and identifiers with letters or hyphens are compared lexically in ASCII sort order. Numeric identifiers always have lower precedence than non-numeric identifiers. A larger set of pre-release fields has a higher precedence than a smaller set, if all of the preceding identifiers are equal. Example: 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta < 1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.

THIS:
>  identifiers consisting of only digits are compared numerically and identifiers with letters or hyphens are compared lexically in ASCII sort order. 

for stealjs we have 2 prerelease version some are called `pre` and the others are called `beta`
with `npm info steal` we get
```     
     '0.16.33',
     '0.16.34',
     '0.16.35',
     '1.0.0-beta.0',
     '1.0.0-beta.1',
     '1.0.0-beta.2',
     '1.0.0-beta.3',
     '1.0.0-pre.0',
     '1.0.0-pre.1',
     '1.0.0-pre.2',
     '1.0.0-pre.3',
     '1.0.0-pre.4',
     '1.0.0-pre.5',
     '1.0.0-pre.6',
     '1.0.0-pre.7'
``` 

`b` is before `p` in the alphabet, thats because we get always the `pre` version

@matthewp i suggest
1. to to create a new pre-release version called `rc` or go ahead with `pre` and forget the `beta` releases
2. remove the `beta` or `pre` tags from npm. saw with `npm dist-tag ls steal`
3. @justinbmeyer we should thought about naming our prereleases. `pre` is usable, but you have to remember that `alpha` or `beta` does not work anymore. the only name can be `rc` which is because the `r` is after the `p` :)

**Comment by matthewp** [comment]
Good points, I think we should just go with the alpha/beta naming going forward to avoid this issue. Very few people installed the pres so few will be affected by this.  Let's just use alpha then beta for in the future.

**Comment by pYr0x** [comment]
@matthewp so do you unpublish the `pre` versions?

**Comment by matthewp** [comment]
Oh, I see, in that case yeah, I guess an `rc` is necessary. Arg, will do that in a minute.

**Comment by matthewp** [comment]
@pYr0x Released 1.0.0-rc.0  Can install with `npm install steal@rc`, instructions on the forums: http://forums.donejs.com/t/stealjs-1-0-beta/316/3
